### PR TITLE
Review

### PR DESCRIPTION
**Pull request by Feketerig**
Changed code review with a collaborator

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 1]
```diff
@@ -30,18 +32,18 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```
[the remark this answers is not in the document]
Comment (NoszDom) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:35]: Wow you're smart!

**Review by NoszDom — COMMENTED** [review]
Nice job!

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -14,6 +14,8 @@
 
   private int torpedoCount = 0;
 
+  private Random random = new Random();
```
[the remark this answers is not in the document]
Comment (Feketerig) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:17]: Thanks.

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Feketerig) [reply src/main/java/hu/bme/mit/spaceship/TorpedoStore.java:35]: Thanks.

**Review by NoszDom — APPROVED** [review]
Approved.

**Pull request merged**